README.md

```diff
@@ -1 +1,32 @@
-# mashov
\ No newline at end of file
+## Steps to generate the forms and email drafts for each employee.
+
+1. Clone this repository.
+
+```
+git clone https://github.com/simon3z/mashov
+```
+
+2. Create a new project in script.google.com. Name it whatever you want. We have to do this because we are adding some variables that point to directories that we are creating for our teams.
+
+3. Copy the Code.gs into the project.
+
+4. Create a directory on Google Drive to save all the forms in.  Get the ID from that directory.
+
+5. Get the ID from the centralized form and spreadsheet.
+
+Google form ID: 1fs7iGd0rbAw1EX5QOmTrYvsHo9OBkNljCKoYbfSf7BA <br>
+Google spreadsheet ID: 1lxaZR2ncek_46x0jNuejyg642_SauAvmqzxCZ9Aw6Fg 
+
+6. Change the variables to meet your needs. See the following for an example (sanitized).
+
+```
+var formFile = DriveApp.getFileById("1fs7iGd0rbvsHo9OBkNljCKoYbfSf7BA"); // Survey form
+var quarterlyFolder = DriveApp.getFolderById("1jsnN-gqqP-g7fW46_DE1Z15I9qJFAZfc"); // Google Drive Folder to place all created survey forms in
+var reviewerSheet = SpreadsheetApp.openById("1lxaZRjNuejyg642_SauAvmqzxCZ9Aw6Fg").getSheetByName("Reviewers"); // Google spreadsheet with list of surveyes per associate
+var viewerEmail = "managers-email-here@redhat.com"; // ATM only editors are supported on google forms, in case it's required should be xxx@redhat.com
+var emailSignature = "-- \n" + "Scott Collier" + "\n";
```

Comment:
I would spell out the required replacements:
1. line 6 - the survey form id
2. line 7 - a google drive to store all the feedback forms
3. line 10 - the review spreadsheet
4. line 15 -  email signature

optional:
5. line 13 - If U want to have visibility to your managers' feedback forms they need to set it up

Reply:
addressed